django/core/management/commands/shell.py

```diff
@@ -7,7 +9,11 @@
 
 
 class Command(BaseCommand):
-    help = "Runs a Python interactive interpreter. Tries to use IPython or bpython, if one of them is available."
+    help = (
+        "Runs a Python interactive interpreter. Tries to use IPython or bpython, if one of them is available"
+        "Like the python CLI, it would read and execute code passed to standard input if any and then exit."
```

Comment:
The 'would' sounds a bit out of place. Also the 'like the python CLI' sounds a little unnecessary now I read it again. Maybe "If there is standard input, it will be immediately executed as code and then the command will exit.".